### ISSUE TITLE
IPv6 query no longer working

### ISSUE THREAD

**Issue by quimic** [andyjsmith/Vultr-Dynamic-DNS#13]
The A record for api6.ipify.org url no longer exists. It looks like one has to use api64.ipify.org instead now.

**Comment by andyjsmith** [comment]
Thanks, switched to a different service. Pretty bad for a public API to just randomly decide to change endpoints. Fixed in 70ec153.

**Comment by quimic** [comment]
Thanks, but this service has the exact same behaviour as ipfy.org: there is no A record for the ip6.seeip.org subdomain (only a AAAA record), which results in the same error if you don't have an IPv6 address:

```
ddns.py:27: UserWarning: Couldn't get IPv6 address: HTTPSConnectionPool(host='ip6.seeip.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105655b40>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
  warnings.warn(f'Couldn\'t get IPv6 address: {str(e)}')
```

**Comment by andyjsmith** [comment]
You're right, giving a warning doesn't make sense, that's from a pull request from someone else. I changed it to just print out that it will only use IPv4. Fixed in 05d9a60.

As for the A vs AAAA record, that's the point of the service, it only has a AAAA record on ip6.seeip.org because it only works on IPv6. That's why the script tries both ip4.* and ip6.*, if your network doesn't use IPv6 it will just skip it and stick to IPv4.
